src/Svg.Skia/SkiaModel.cs

```diff
@@ -502,18 +522,27 @@ public SkiaSharp.SKColorChannel ToSKColorChannel(SKColorChannel colorChannel)
                     return null;
                 }
 
-                return SkiaSharp.SKImageFilter.CreateColorFilter(
-                    ToSKColorFilter(colorFilterImageFilter.ColorFilter),
-                    ToSKImageFilter(colorFilterImageFilter.Input),
-                    ToCropRect(colorFilterImageFilter.Clip));
+                return colorFilterImageFilter.Clip is { } clip
+                    ? SkiaSharp.SKImageFilter.CreateColorFilter(
+                        ToSKColorFilter(colorFilterImageFilter.ColorFilter),
+                        ToSKImageFilter(colorFilterImageFilter.Input),
+                        ToSKRect(clip.Rect))
+                    : SkiaSharp.SKImageFilter.CreateColorFilter(
+                        ToSKColorFilter(colorFilterImageFilter.ColorFilter),
+                        ToSKImageFilter(colorFilterImageFilter.Input));
```

Comment:
This is a bit unfortunate, but to avoid ambiguity, I could not make the SKRect parameter nullable. However, this code is not too painful. The main change here is that we now just use the Rect directly instead of wrapping it in a CropRect. 

SkiaSharp 3.x no longer has the crop rect type at all, so not much we can do besides adding a dummy type - not so nice.

Note: I did not touch the code in the source gen as I am not sure if that generated code is meant to work with multiple versions of skia? If a new source gen is supposed to work with older 2.x versions, then the new APIs will not exist. However, I can also add those changes if need be.